vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1560,6 +1651,108 @@ void vtkF3DRenderer::ResetCameraClippingRange()
   this->GridActor->SetUseBounds(gridUseBounds);
 }
 
+vtkBoundingBox vtkF3DRenderer::ComputeVisiblePropOrientedBounds(const vtkMatrix4x4* matrix)
+{
+  const auto isMatrixAxisAligned = [](const vtkMatrix4x4* m, const double tol = 1e-8)
```

Comment:
Can't you just check that the sum of the asbolute value of all the elements (L1 norm) of the matrix is exactly 4? A flipped matrix would introduce a higher value I believe.

Reply:
Maybe, I'm not sure TBH, I guess we would still be ok if there was some scaling in addition to swapping axes in the general case? But if we trust we're only using normalized vectors to build the matrix then your suggestion would work.

Reply:
fair enough.